ui/desktop/src/ChatWindow.tsx

```diff
@@ -173,8 +179,6 @@ function ChatContent({
 
         <Input
           handleSubmit={handleSubmit}
-          handleInputChange={handleInputChange}
-          input={input}
```

Comment:
if you're removing this, you likely don't need the `handleChange` at all in Input.tsx any more?

It's no longer "controlled input" https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable

Reply:
Input needs it to accumulate the chars (if I get rid of it - won't work) IIUC?